src/server/accounting.c

```diff
@@ -264,54 +227,128 @@ sum_resc_alloc(job *pjob, pbs_list_head *list)
+			} else
+				temp[dest_index++] = value[value_index];
+		}
+		if (need_quotes) {
+			if ((value[0] != '\"' && value[len-1] != '\"') ||
+			    ((value[0] == '\"' && value[len-1] != '\"') ||
+			    (value[0] != '\"' && value[len-1] == '\"'))) {
+				temp[0] = '\"';
+				temp[dest_index++] = '\"';
+				temp[dest_index] = '\0';
+				if (!pbs_strcat(buf, buf_size, fmt)) {
+					log_err(errno, __func__, "Failed to allocate memory."); 
+					return ret;
+				}
+				if (!pbs_strcat(buf, buf_size, temp)) {
+					log_err(errno, __func__, "Failed to allocate memory.");
+					return ret;
+				}
+			} else {
+                        	temp[dest_index] = '\0';
+                        	if (!pbs_strcat(buf, buf_size, fmt)) {
+					log_err(errno, __func__, "Failed to allocate memory.");
+                                	return ret;
+				}
+                        	if (!pbs_strcat(buf, buf_size, &temp[1])) {
+					log_err(errno, __func__, "Failed to allocate memory.");
+                                	return ret;
+				}
+                	}
+		} else {
+			temp[dest_index] = '\0';
+			if (!pbs_strcat(buf, buf_size, fmt)) {
+				log_err(errno, __func__, "Failed to allocate memory.");
+				return ret;
+			}
+			if (!pbs_strcat(buf, buf_size, &temp[1])) {
+				log_err(errno, __func__, "Failed to allocate memory.");
+				return ret;
+			}
+		}
+		ret = 0;
 	}
+	return ret;
 }
 
-/* These are various printing formats used in acct_job() */
-#define	GRIDNAME_FMT		"gridname=\"%s\" "
-#define USER_FMT		"user=%s "
-#define	GROUP_FMT		"group=%s "
-#define	ACCOUNT_FMT		"account=\"%s\" "
-#define	PROJECT_FMT1		"project=\"%s\" "
-#define	PROJECT_FMT2		"project=%s "
-#define	ACCOUNTING_ID_FMT	"accounting_id=\"%s\" "
-#define	JOBNAME_FMT		"jobname=%s "
-#define	QUEUE_FMT		"queue=%s "
-#define	RESVNAME_FMT		"resvname=%s "
-#define	RESVID_FMT		"resvID=%s "
-#define	RESVJOBID_FMT		"resvjobID=%s "
-#define	ARRAY_INDICES_FMT	"array_indices=%s "
-#define	EXEC_HOST_FMT		"exec_host=%s "
-#define	EXEC_VNODE_FMT		"exec_vnode=%s "
+/* These are various printing formats used in acct_job() and acct_resv() */
+#define	GRIDNAME_FMT		"gridname="
+#define	USER_FMT		"user="
+#define	GROUP_FMT		" group="
+#define	ACCOUNT_FMT		" account="
+#define	PROJECT_FMT		" project="
+#define	ACCOUNTING_ID_FMT	" accounting_id="
+#define	JOBNAME_FMT		" jobname="
+#define	QUEUE_FMT		" queue="
+#define	RESVNAME_FMT		" resvname="
+#define	RESVID_FMT		" resvID="
+#define	RESVJOBID_FMT		" resvjobID="
+#define	ARRAY_INDICES_FMT	" array_indices="
+#define	EXEC_HOST_FMT		" exec_host="
+#define	EXEC_VNODE_FMT		" exec_vnode="
+#define	RESV_OWNER_FMT		" owner="
+#define	RESV_NAME_FMT		" name="
+#define	RESV_NODES_FMT		" nodes="
+#define	JOB_ALTID_FMT		" alt_id="
+#define	PROV_NODE_FMT		" provision_vnode="
+#define	RUNCOUNT_FMT		" run_count="
 
 /* Amount of space needed in account log buffer for the ctime, qtime, etime, */
 /* start attributes */
-#define ACCTBUF_TIMES_NEED	72
-
+#define ACCTBUF_TIMES_NEED	90
```

Comment:
I calculated  that to store all the *time attribute in the accounting log it was consuming 69 and the defined size was 72 I thought what if values increases in any case. But now I guess it will not increase and it will be better to keep it 72 I'll make the change.

Reply:
Yes, its in microseconds.